internal/controller/forwarding/forwarding_controller.go

```diff
@@ -132,12 +133,30 @@ func (r *ReconcileForwarder) updateStatus(instance *logging.ClusterLogForwarder)
 	return periodicRequeue, nil
 }
 
+// AnnotateLoggingClusterLogForwarders gets a list of existing clusterlogforwarders.logging.openshift.io
+// and annotates them with "logging.openshift.io/needs-migration"
+func AnnotateLoggingClusterLogForwarders(k8sClient client.Client, apiReader client.Reader) error {
+	var clfList logging.ClusterLogForwarderList
+	if err := apiReader.List(context.TODO(), &clfList); err != nil {
+		return err
+	}
+	if len(clfList.Items) > 0 {
```

Comment:
Is this check redundant since the for loop will be skipped if clfList.Items is empty?